mage/template.go

```diff
@@ -108,3 +108,48 @@ func list() error {
 }
 
 `
+
+var mageTpl = `// +build mage
+
+package main
+
+import (
+	"os/exec"
+	"fmt"
+	"github.com/magefile/mage/mg" // mg contains helpful utility functions, like Deps
+)
+
+// Default target to run when none is specified
+// var Default = Build
+
+// A build step that requires additional params, or platform specific steps for example
+func Build() error {
+	mg.Deps(InstallDeps)
+	fmt.Println("Building...")
+	cmd := exec.Command("go", "build", "-o", "MyApp", ".")
+	return cmd.Run()
+}
+
+// A custom install step if you need your bin someplace other than go/bin
+func Install() error {
+	mg.Deps(Build)
+	fmt.Println("Installing...")
+	cmd := exec.Command("echo", "Pretend to copy someplace else")
```

Comment:
can you make this os.Rename("./MyApp",  "/usr/bin/MyApp") or something? It's nice to show that you don't have to drop down to running external commands to get stuff done.